src/Xamarin.Android.Build.Tasks/Tasks/GenerateJavaStubs.cs

```diff
@@ -272,6 +271,7 @@ void Run (DirectoryAssemblyResolver res, bool useMarshalMethods)
 
 			using (var acw_map = MemoryStreamPool.Shared.CreateStreamWriter ()) {
 				foreach (TypeDefinition type in javaTypes) {
+					string mvid = type.Module.Mvid.ToString ();
```

Comment:
This does `Guid.ToString()` for every type, can we move this a few lines down, so it only happens when there is a conflict?